composer.json

```diff
@@ -199,6 +199,7 @@
         "git-hook-install": "@php .github/contrib/install.php",
         "git-hook-uninstall": "@php .github/contrib/uninstall.php",
         "integration-behaviour-tests": [
+            "@composer create-test-db",
```

Comment:
I didn't pay attention to this fix required to initialize the test database on first run.
Should I move this fix into a new PR ?